### PR TITLE
repoquery: use new api methods duplicated and latest

### PR DESCRIPTION
**Pull request by mluscon**

### CODE REVIEW

File: plugins/repoquery.py [hunk 1]
```diff
@@ -293,16 +291,20 @@ def run(self, args):
...
+            q.autoremove(self.base.sack, self.base.yumdb)
+        elif self.opts.list:
+            q = getattr(q, self.opts.list)()
 
         if self.opts.pkgfilter == "duplicated":
-            dups = dnf.query.duplicated_pkgs(q, self.base.conf.installonlypkgs)
-            q = q.filter(pkg=dups)
+            installonly = self.installonly(q)
+            exclude = [pkg.name for pkg in installonly]
+            q = q.duplicated(exclude)
```
Comment (jsilhan) [inline plugins/repoquery.py]: q = q.filter(pkg_not=list(self.installonly(q))).duplicated()
see https://github.com/rpm-software-management/dnf/pull/367/files#r43499096
(I am not sure whether pkg__not works now)

Comment (jsilhan) [inline plugins/repoquery.py]: your commit actually fixes [1], reference this in the message, please
[1] https://bugzilla.redhat.com/show_bug.cgi?id=1231572

Comment (mluscon) [inline plugins/repoquery.py]: done

**Comment by jsilhan** [comment]
Merged, thanks

**Pull request closed without merging**